src/components/auth/signup/signup-design.html

```diff
@@ -0,0 +1,59 @@
+
+<div ng-controller="@A.2">
```

Comment:
The `ng-controller` is not necessary because I using `angular-ui-router` that define a state and bind a controller.
please reference it: https://github.com/inwinstack/s3-portal-ui/blob/dev/src%2Fcomponents%2Fauth%2Fsignup%2Fsignup.js#L8